_episodes/11-using-arrays-transformations.md

```diff
@@ -60,7 +60,7 @@ Taking the above example again, this would result in a string with the days of t
 >1. Make sure you have already split the author names into individual cells using ```Edit cells->Split multi-valued cells``` (you should have done this in the Clustering lesson)
 >2. On the Authors column, use the dropdown menu and select ```Facet->Custom text facet...```
 >3. The ```Custom text``` facet function allows you to write GREL functions to create a facet
->4. In the Expression box type ```value.contains(",").toString()```
+>4. In the Expression box type ```value.contains(",")```
```

Comment:
Removing the 'toString()' here will prevent the lesson working with OpenRefine 3.1. Leaving it in makes the instruction work in all versions of OpenRefine. I don't have an issue with removing it but just noting in case you think it's worth leaving in

Reply:
Hm, true… However, that version is [almost 4 years old](https://github.com/OpenRefine/OpenRefine/releases/tag/3.1), and at least 2 security vulnerabilities were fixed since then.

I'd argue that one more nudge to get people to update is OK.